testsuite/classlibrary/TestUnitTest.sc

```diff
@@ -1,30 +1,15 @@
 TestUnitTest : UnitTest {
 
-	var someVar, toreDown, count=0;
+	var someVar;
 
 	setUp {
 		someVar = \setUp;
-		count = count + 1;
-	}
-	tearDown {
-		someVar = \tearDown;
-		toreDown = true;
 	}
 
 	test_setUp {
-		this.assertEquals(count, 1, "count should be on 1");
 		this.assertEquals(someVar, \setUp, "someVar be set in setUp");
 	}
 
-	test_toreDown{
-		this.assertEquals(toreDown, true, "toreDown should be set at the end of the last test");
```

Comment:
i'm missing some context here. how did this ever pass in the first place?

Reply:
`toreDown = true` happens during the `tearDown` after `test_setUp`.